pom.xml

```diff
@@ -34,10 +34,11 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.javadoc.skip>true</maven.javadoc.skip>
-        <jenkins.version>1.642.3</jenkins.version>
-        <jenkins-test-harness.version>2.18</jenkins-test-harness.version>
+        <jenkins.version>2.7.3</jenkins.version>
```

Comment:
Do you want to make `Jenkinsfile` test against 2.102 also?

Reply:
Nope. It blows up on enforcer